authors/apps/authentication/views.py

```diff
@@ -1,9 +1,11 @@
+import jwt
 from rest_framework import status
 from rest_framework.generics import RetrieveUpdateAPIView
 from rest_framework.permissions import AllowAny, IsAuthenticated
 from rest_framework.response import Response
 from rest_framework.views import APIView
 
+from authors.settings import SECRET_KEY
```

Comment:
@hoslack the secret key has been imported here from the settings file. I do not think @davenjeru needs to create a .env file for that.